### PR TITLE
Chatterino 2.4.4

### PR DESCRIPTION
**Pull request by ktnjared**
Chatterino 2.4.4 for request #22593
- Manifest validated locally
- Sandbox validated
- The previous blocker on #22593 was addressed by the Chatterino team: vcdist installation is now handled by the package Installer.

--

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109519&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/c/Chatterino/Chatterino/2.4.4/Chatterino.Chatterino.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Chatterino.Chatterino
+PackageVersion: 2.4.4
+InstallerLocale: en-US
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Chatterino2/chatterino.exe
+  PortableCommandAlias: Chatterino2/chatterino.exe
+Scope: user
```
Comment (ItzLevvie) [inline manifests/c/Chatterino/Chatterino/2.4.4/Chatterino.Chatterino.installer.yaml:11]:
```suggestion
```

File: manifests/c/Chatterino/Chatterino/2.4.4/Chatterino.Chatterino.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-05-13
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/Chatterino/chatterino2/releases/download/v2.4.4/Chatterino.Installer.exe
+  InstallerSha256: 0B3A2BAC515499E4F90C2844E67EB449A1437302E54B4B8B1EDA976806F16423
```
Comment (ItzLevvie) [inline manifests/c/Chatterino/Chatterino/2.4.4/Chatterino.Chatterino.installer.yaml:22]:
```suggestion
- Architecture: x86
  InstallerType: inno
  InstallerUrl: https://github.com/Chatterino/chatterino2/releases/download/v2.4.4/Chatterino.Installer.exe
  InstallerSha256: 0B3A2BAC515499E4F90C2844E67EB449A1437302E54B4B8B1EDA976806F16423
  Scope: user
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  InstallerType: inno
  InstallerUrl: https://github.com/Chatterino/chatterino2/releases/download/v2.4.4/Chatterino.Installer.exe
  InstallerSha256: 0B3A2BAC515499E4F90C2844E67EB449A1437302E54B4B8B1EDA976806F16423
  Scope: machine
  InstallerSwitches:
    Custom: /ALLUSERS
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/c/Chatterino/Chatterino/2.4.4/Chatterino.Chatterino.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-05-13
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/Chatterino/chatterino2/releases/download/v2.4.4/Chatterino.Installer.exe
```
Comment (Felanbird) [inline manifests/c/Chatterino/Chatterino/2.4.4/Chatterino.Chatterino.installer.yaml:21]: I just want to note that the installer from Github releases is just a re-upload of the one listed on the website, hosted on digitalocean.

```suggestion
  InstallerUrl: https://chatterino.fra1.digitaloceanspaces.com/bin/2.4.4/Chatterino%20Installer.exe
```

**Comment by ItzLevvie** [comment]
[Policy] Interactive-Only-Installer

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by ItzLevvie** [comment]
image

**Comment by pajlada** [comment]
@ktnjared v2.4.5 has been released now, using an updated installer. The new installer URL is `https://github.com/Chatterino/chatterino2/releases/download/v2.4.5/Chatterino.Installer.exe` and its sha256 is `60fbcd77d637b529e0a056e7c743c819e1bdb5ab376c0da27b6df3f167ca8767`

**Pull request closed without merging**